pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-email-actions-and-events.md

```diff
@@ -0,0 +1,528 @@
+---
+$title: Actions and events
+order: 7
+formats:
+  - email
+teaser:
+  text: >-
+    The on attribute is used to install event handlers on elements. The events
+    that are supported depend on the element.
+---
+
+<!--
+This file is imported from https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md.
+Please do not change this file.
+If you have found a bug or an issue please
+have a look and request a pull request there.
+-->
+
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+
+
+
```

Comment:
Are these many extra lines necessary?

Reply:
I've removed them, we'll see if prettifier puts them back!